src/app/application/routes.ts

```diff
@@ -0,0 +1,16 @@
+import type { Component } from 'vue'
+import type { RouteRecordRaw } from 'vue-router'
+
+export const routes = (notFoundViews: (() => Promise<Component>)[]): RouteRecordRaw[] => {
+  return [
+    {
+      path: '/404',
+      name: 'not-found',
+      alias: '/:pathMatch(.*)*',
+      meta: {
+        title: 'Item not found',
+      },
+      component: () => notFoundViews[notFoundViews.length - 1](),
```

Comment:
I wanted to point this one out. Its not super important as its a non important 404 page, but I wanted to using labelling to specify the component for this from The Outside, as labels are mostly arrays, this is why we do the weird "get the last one in the array" thing. If we want to overwrite the 404 page at anypoint this "ge tthe last one of the labelled things" should"Just Work(tm)" for us.